### PR TITLE
Add two proposals -- unfreeze voter rewards, unfreeze celo gold transfers

### PR DESCRIPTION
**Pull request by aslawson**
Add two proposals using the new CGP Template
- unfreeze voter rewards
- unfreeze celo gold transfers

### CODE REVIEW

File: CGPs/0001.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - Data: 160000000000000000000 (0.00016 * 10^24)
+  - Value: 0 (NA)
+2. Set Target Voting Adjustment Factor (and keep max the same)
+  - Destination: EpochRewards, setTargetVotingYieldParameters
+  - Data: max = 500000000000000000000 (0.0005 * 10^24), adjustmentFactor = 273972602740000000000 (0.00027397260274 * 10^24)
+  - Value: 0 (NA)
+
+## Verification
+
+- Check that voting accounts receive funds after one epoch
```
[the remark this answers is not in the document]
Comment (asaj) [reply CGPs/0001.md:33]: This would include CLI commands and code pointers

[the remark this answers is not in the document]
Comment (asaj) [reply CGPs/0001.md:33]: Perhaps this section should be renamed "How to verify", wdyt?

File: CGPs/0001.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# CGP [0001]: Unfreeze Voter Rewards
+
+- Date: 2020-05-12
+- Author: @aslawson
+- Status: DRAFT
+- Governance Propsal ID #: [if submitted]
```
[the remark this answers is not in the document]
Comment (aslawson) [reply CGPs/0001.md:6]: oh no -- I had copied this before fixing the other PR

File: CGPs/0001.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- Check that voting accounts receive funds after one epoch
+
+## Risks
+
+Low risk
+
+## Useful Links
+
+- NA
```
[the remark this answers is not in the document]
Comment (aslawson) [reply CGPs/0001.md:41]: good idea

File: CGPs/0001.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (aslawson) [reply CGPs/0001.md]: I see -- I think the title is ok.  I'll just tweak the explanation to "An explanation of how **voters** can verify"

File: CGPs/0001.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  - Data: 160000000000000000000 (0.00016 * 10^24)
+  - Value: 0 (NA)
+2. Set Target Voting Adjustment Factor (and keep max the same)
+  - Destination: EpochRewards, setTargetVotingYieldParameters
+  - Data: max = 500000000000000000000 (0.0005 * 10^24), adjustmentFactor = 273972602740000000000 (0.00027397260274 * 10^24)
+  - Value: 0 (NA)
+
+## Verification
+
+- Methods called:
```
Comment (asaj) [inline CGPs/0001.md:33]: I think this section should have step-by-step "run these commands to verify this proposal does what I say it does" instructions

File: CGPs/0002.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+## Proposed Changes
+
+1. Remove GoldToken smart contract from the Freezer
+  - Destination: Freezer, unfreeze
+  - Data: 0x471EcE3750Da237f93B8E339c536989b8978a438 (GoldToken Address)
+  - Value: 0 (NA)
+
+## Verification
+
+- Methods called:
```
Comment (asaj) [inline CGPs/0002.md:22]: Ditto

File: CGPs/0001.md (hunk 4 above)
Comment (asaj) [reply CGPs/0001.md:33]: If your plan was to add this after the proposal was submitted, please leave a comment indicating this

Comment (aslawson) [reply CGPs/0001.md:33]: I feel like it does -- it directs to the exact CeloCli commands to run and what to expect.  I could only make it more exact if I gave them a specific account voting account (eg my own)...

For checking the parameter value, you mean something like guiding them through using the truffle console to access the smart contract variables?

**Pull request merged**